devices/milightRF24.coffee

```diff
@@ -78,7 +92,28 @@ module.exports = (env) ->
           button = Buttons.Group4Off
       
       @gateway.sendButton(id, zone, button, false)
+      
+      @_loop(id, zone, button, false, 0, 0, 0,0,0)
+  
+    _loop: (id, zone, button, longPress, discoMode, brightness, r, g, b) ->
+      dataObj =
+        raw: "loop",
+        id: id,
+        zone: zone,
+        button: button,
+        longPress: longPress,
+        discoMode: discoMode,
+        brightness: brightness,
+        color:
+          r: r,
+          g: g,
+          b: b
+          
+      @.emit("ReceivedData", dataObj);
     
+  MilightRF24::__proto__ = events.EventEmitter.prototype
```

Comment:
Why you do it that way?

Wouldn't `class MilightRF24 extends  events.EventEmitter` have the same effect?